assignment/tool/src/webapp/js/assignments.js

```diff
@@ -1006,6 +1006,92 @@ ASN.disableTimesheetSetupSection = function()
     ASN.toggleAutoAnnounceEstimate(false);
 }
 
+ASN.autocompleteRubricWithSelfReport = function(event) {
```

Comment:
As this is a brand new function, you should probably use a two space indentation. It's what we use in all the webcomponents and what we specify in our eslint config.